chia/full_node/mempool_check_conditions.py

```diff
@@ -42,7 +41,7 @@ def get_name_puzzle_conditions(
         # conditions must use integers in canonical encoding (i.e. no redundant
```

Comment:
we should remove this comment too

Reply:
actually, this comment is still valid. The flag that was removed (`COND_CANON_INTS`) is now implied. With the soft-fork, integers are *required* to be in canoical form (i.e. no redundant leading zeroes).

Removing this flag, and making its behavior mandatory, was one of the simplifications in `chia_rs` I made recently.